packages/catalog-process/src/services/catalogService.ts

```diff
@@ -385,6 +424,221 @@ function isTenantInSignalHubWhitelist(
     : false;
 }
 
+async function innerCreateEService(
+  seed: {
+    eServiceSeed: catalogApi.EServiceSeed;
+    eServiceTemplateReferences?: EServiceTemplateReferences;
```

Comment:
Let's avoid optional fields in favor of this approach to force callers to explicitly set a value

```suggestion
    eServiceTemplateReferences: EServiceTemplateReferences | undefined;
```